Userland/Libraries/LibVirtGPU/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+set(SOURCES
+    Device.cpp
+    Image.cpp
+    Shader.cpp
+)
+
+serenity_lib(LibVirtGPU virtgpu)
+target_link_libraries(LibVirtGPU PRIVATE LibCore)
+target_sources(LibSoftGPU PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/../LibGPU/Image.cpp")
```

Comment:
Why are we adding sources to LibSoftGPU here in LibVirtGPU/CMakeLists.txt in commit `LibVirtGPU: Add a new GPU device that talks to our VirtIO-GPU driver`?

Reply:
Oops, rebase gone wrong